package.json

```diff
@@ -23,6 +23,7 @@
     "@types/mocha": "^9.0.0",
     "encoding": "^0.1.13",
     "mocha": "^9.0.3",
+    "prettier": "2.7.1",
```

Comment:
It seems that the dependency `prettier` is not necessary. Would you mind to double-check on this?

Reply:
Hey @ironaddicteddog, sp I was going off their recommendations for setting up Prettier here: [https://prettier.io/docs/en/install.html](https://prettier.io/docs/en/install.html)

> Note: If you forget to install Prettier first, npx will temporarily download the latest version. That’s not a good idea when using Prettier, because we change how code is formatted in each release! It’s important to have a locked down version of Prettier in your package.json. And it’s faster, too.

Reply:
They reiterrate this a few times, specifically:

> Note: Don’t skip the regular local install! Editor plugins will pick up your local version of Prettier, making sure you use the correct version in every project. (You wouldn’t want your editor accidentally causing lots of changes because it’s using a newer version of Prettier than your project!)

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.

Reply:
Noted, thank you